x-pack/plugins/observability/public/hooks/slo/use_clone_slo.ts

```diff
@@ -40,7 +41,7 @@ export function useCloneSlo() {
 
         const optimisticUpdate = {
           ...data,
-          results: [...(data?.results || []), { ...sloUsedToClone, name: slo.name }],
+          results: [...(data?.results || []), { ...sloUsedToClone, name: slo.name, id: uuidv1() }],
```

Comment:
🐛 I noticed this bug while working on this PR, since it's a small fix I included it here. Without specifying the id, when we were cloning an SLO, we were getting an extra identical SLO and we needed to refresh the page to get ride of it.
By providing a new id, the optimistic slo is "removed" correctly when the success handler is executed.